cita-executor/core/src/contracts/solc/sys_config.rs

```diff
@@ -341,7 +341,7 @@ impl<'a> SysConfig<'a> {
     }
 
     pub fn default_auto_exec() -> bool {
-        error!("Use the default autoEXEC.");
+        debug!("Use the default autoEXEC.");
```

Comment:
`Use the default auto exec config` ?

Reply:
Some users upgrade CITA from v0.19 t0 v0.23, and there is no `getAutoExec()` in  genesis. So it uses default config.